Gemfile

```diff
@@ -45,6 +45,8 @@ gem 'view_component'
 
 # DLSS/domain-specific dependencies
 gem 'cocina-models'
+gem 'dor-services-client', '>= 15.2.1'
+gem 'dor-workflow-client', '>= 7.6.1'
```

Comment:
Any reason not to be a bit more pessimistic, in case new major versions break things?

Reply:
The bigger problem we've been having is apps not getting upgraded because they're pinned.

In general, since we control all of the apps and these are all API clients, we tend to just change the APIs, change the clients, and update the apps without worrying about backwards compatibility. Well, at least that is my perspective.

Reply:
@sul-dlss/infrastructure-team happy to discuss this further and if necessary amend with additional PR.